authors/apps/authentication/tests/test_reset_password.py

```diff
@@ -0,0 +1,85 @@
+import json
+
+from django.test import TestCase
+from rest_framework.test import APIClient
+from rest_framework import status
+from django.core import mail
+
+
+class PasswordTestCase(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.forgot_password = {
+            'email': 'remmy@test.com'
+        }
+
+        self.reset_password = {
+            'email': 'remmy@test.com',
+            'token': '1234567890',
+            'new_password': 'Password123'
+        }
+
+        self.renew_password = {
+            'email': 'remmy@test.com',
+            'old': 'password1',
+            'new_password': 'Password123'
+        }
+
+        self.old_password = {
+            'email': 'remmy@test.com',
+            'old': 'password1'
+        }
+
+    def test_forgot_password(self):
+        ''' Test that checks if a user receives a reset password mail'''
+
+        response = self.client.post(
+            'api/users/forgot_password',
```

Comment:
Django did not raise issues with this (and the others) URLs? You literally don't have any modifications in urls.py?